### ISSUE TITLE
Getting ForbiddenAttributesError during search on TERRA-REF instance

### ISSUE THREAD

**Issue by craig-willis** [PecanProject/bety#582]
As discussed, I'm getting a ForbiddenAttributesError when searching the TERRA-REF BETY via API.

Request URL:
```
https://terraref.ncsa.illinois.edu/bety/api/v1/search?method_name=Canopy%20Cover%20Estimation%20from%20RGB%20images&trait=canopy_cover&key=xxx
```

Output:
```
{
  "metadata": {
    "URI": "https://terraref.ncsa.illinois.edu/bety/api/v1/search?method_name=Canopy%20Cover%20Estimation%20from%20RGB%20images&trait=canopy_cover&key=GZJZWnJpnDBhmKk7k6vb46z6lW6vjxSniRivRl2I",
    "timestamp": "2018-04-25T15:59:44.042-05:00",
    "count": 0
  },
  "errors": "ActiveModel::ForbiddenAttributesError",
  "data": [

  ]
}
```

Expected output:
* Actual search results

**Comment by dlebauer** [comment]
looks like this has been fixed and deployed on terraref.ncsa.illinois.edu/bety

**Comment by gsrohde** [comment]
Fixed by release 5.0.  Closing.
